sections/introduction.tex

```diff
@@ -0,0 +1,61 @@
+PageRank\cite{pagerank_1999} is a ranking algorithm, which is based on the random walk process.
+The principle is to let a number of surfers walk between nodes in a graph, where each surfer takes one step in each iteration.
+After an appropriate number of steps, the number of surfers on each node will be the ranking of the given node, where more surfers give a higher rank.
+The edges in the graph can be based on many interactions, such as hyperlinks, similarities, genres, etc.
+
+In \cite{yang2009topic} \citeauthor{yang2009topic} investigate how to combine topic model \gls{lda}, \gls{lm}, and PageRank.
+They also investigate a modified version of the PageRank algorithm which takes topic-level information into account.
+They use a scientific article dataset where they manually label 200 articles from their dataset based on related queries. 
+In this paper, we want to expand the framework which \cite{yang2009topic} have provided.
+However, the data set which \cite{yang2009topic} uses is based on scientific articles, which includes citation to other similar articles.
+This citation scheme does not necessarily extent to other document types, such as news articles.
+
+In this paper, we want to investigate whether using a similarity measure based on the topics generated by a topic model can be used with PageRank.
+
+The combination of \gls{lda} and PageRank is interesting, since underlying document topics can yield more information than usually given by a single document.
+These topics can be seen as a set of documents sharing a specific topic. 
+
+When evaluating search engines, manual labeling of documents is usually used to measure the performance of models\cite{yang2009topic}\cite{Tang2008}.
+Since the topics, that are created by the \gls{lda}, are distributions of topics in documents, we can use these to evaluate a search algorithm's retrieval performance. 
+
+Our goal is thus to improve news article search by using a combination of \gls{lda}, \gls{lm}, and PageRank.
+For our experiments we use the Nordjyske database of news articles as our dataset.
+We also want to investigate whether an automatic evaluation measure can be used to evaluate a search engine.
+Here we want to look into whether it makes sense to make an evaluation metric based on whether search results are in a given topic.
+Finally, since the news article dataset does not include connections between the articles, how PageRank can be run on this needs to be investigated.
+With these goals we look at the following questions:
+
+\begin{quote}
+	\emph{How can information retrieval techniques be evaluated based on search queries?}
+	\begin{itemize}
+		\item \emph{How can queries be generated for a dataset?}
+		\item \emph{How can evaluation be done in a way that favors abstraction, rather than word frequency?}
+	\end{itemize}
+\end{quote}
+\vspace{0.1 cm}
+
+\begin{quote}
+	\emph{How can PageRank be used on a document dataset with no connections between documents?}
+\end{quote}
```

Comment:
Jeg vil gerne lave dette til noget mere formal som vi kan referere, og mere til goals end spørgsmål.
Derudover kan jeg ikke så godt lide det første spørgsmål da det virker for generelt, som om vi har tænkt os at undersøge en masse evaluation metrics i dette paper.
```suggestion
These goals are expressed in \todo{ref goals}.

\begin{quote}
	\begin{itemize}
		\item \emph{How can we generate queries for a dataset to use for \gls{ir}?}
		\item \emph{How can evaluate \gls{ir} methods in a way that favors abstraction, rather than word frequency?}
	        \item \emph{How can we contruct an adjacency matrix based on a dataset with no explitcit connections between documents, to be able to use PageRank ?}
	\end{itemize}
\end{quote}
\todo[inline]{make this into its own goal class that is referenceable like with definitions.}
```
btw, ved godt at det var mig der tidligere ville fjerne 'explicit'. Det lyder bare godt.